internal/convenience/repository/voucher_repository_test.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/calindra/nonodo/internal/commons"
 	"github.com/calindra/nonodo/internal/convenience/model"
```

Comment:
We are importing the same package twice here

Reply:
fixed and found another instance of same mistake